catalogs/catalog-jdbc-common/src/main/java/com/datastrato/gravitino/catalog/jdbc/operation/JdbcTableOperations.java

```diff
@@ -117,18 +118,37 @@ public List<String> listTables(String databaseName) throws NoSuchSchemaException
 
   @Override
   public JdbcTable load(String databaseName, String tableName) throws NoSuchTableException {
+    // We should handle case sensitivity and wild card issue in some catalog tables, take a
+    // MySQL table for example.
+    // 1. MySQL will get table 'a_b' and 'A_B' when we query 'a_b' in a case-insensitive charset
+    // like utf8mb4.
+    // 2. MySQL will view 'a_b' as a wild card, and it will match any table name that starts with
+    // 'a',then any character and then 'b'.
```

Comment:
FYI:
"MySQL treats 'a_b' as a wildcard, matching any table name that begins with 'a', followed by any character, and ending with 'b'. "

Reply:
fixed